### DISCUSSION TITLE
Date and timestamp of a fike

### DISCUSSION THREAD

**Discussion by donzefe** [TeamAmaze/AmazeFileManager#4285] [Ideas]
Hi

I like the App, thank you!

A thing nasty for me:
When copying or moving a file, the date- and timestamp of the file is not kept. In my opinion, the date- and timestamp should be kept as it was in the original file/folder!
(reason is that my syncing and backup Apps sometimes habe Problems after... Thats for me the one and only reason to NOT using this App :-(  )

Would be great uf this could be adapted!
Best regards

**Comment by EmmanuelMess** [comment]
This is managed by Linux, not by Amaze. Even if we can hack a solution modifying the metadata, it would be problematic for users expecting the default OS behaviour.

**Reply by donzefe** [reply]
Thanks for your answer! This seems strange to me, but I believe you.  Ok.
Have a nice day and thanks for your work!
